CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ the code was deployed.
 
 ## [Unreleased]
 ### Added
+- Deployment instructions in the README file.
```

Comment:
Can you please add the ClickUp number here?

Reply:
Oops, somehow I was convinced there wasn't a ticket.